vng_api_common/authorizations/serializers.py

```diff
@@ -0,0 +1,94 @@
+import logging
+
+from django.db import transaction
+from django.utils.translation import ugettext_lazy as _
+
+from rest_framework import serializers
+
+from vng_api_common.authorizations.models import Applicatie, Autorisatie
+
+logger = logging.getLogger(__name__)
+
+
+class AutorisatieSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Autorisatie
+        fields = (
+            'component',
+            'zaaktype',
+            'scopes',
```

Comment:
purely aesthetic, but can you swap the order of `scopes` and `zaaktype`?

The reason is that `zaaktype` and `max_vertrouwelijkheidaanduiding` actually will depend on the value of `component`, while `component` and `scopes` are the 'constant' fields.

I'll need to figure out how to cleverly generate an API spec from that, which is a problem for Future Me :smile:

Reply:
Changed the order )